web/src/beta/features/Editor/index.tsx

```diff
@@ -224,21 +216,14 @@ const Editor: React.FC<Props> = ({ sceneId, projectId, workspaceId, tab }) => {
                   gutter="start"
                   initialSize={metrics.bottomPanelMinHeight}
                   minSize={metrics.bottomPanelMinHeight}
-                  maxSize={metrics.bottomPanelMaxHeight}>
+                  maxSize={metrics.bottomPanelMaxHeight}
+                  localStorageKey="bottomPanel">
                   {bottomPanel}
                 </Resizable>
               )}
             </CenterContent>
           </Center>
-          {rightPanel && (
-            <Resizable
-              direction="vertical"
-              gutter="start"
-              initialSize={metrics.propertyMenuWidth}
-              minSize={metrics.propertyMenuMinWidth}>
-              {rightPanel}
-            </Resizable>
-          )}
+          {rightPanel && rightPanel}
```

Comment:
Just do {rightPanel} and {leftPanel} in this file, and then in the two panel's just return null instead of undefined and it should be fine 👍🏻